exe-nix-buildkite/Main.hs

```diff
@@ -42,14 +42,16 @@ import System.Process hiding ( env )
 import Data.Text ( pack, unpack )
 import Data.Text.IO ( readFile )
 
+-- sort
+import Data.Sort ( uniqueSort )
 
 main :: IO ()
 main = do
   jobsExpr <- fromMaybe "./jobs.nix" . listToMaybe <$> getArgs
 
   -- Run nix-instantiate on the jobs expression to instantiate .drvs for all
   -- things that may need to be built.
-  inputDrvPaths <- Prelude.lines <$> readProcess "nix-instantiate" [ jobsExpr ] ""
+  inputDrvPaths <- uniqueSort <$> Prelude.lines <$> readProcess "nix-instantiate" [ jobsExpr ] ""
```

Comment:
I'm curious if you chose `uniqueSort` specifically? I would lean slightly more towards https://hackage.haskell.org/package/containers-0.6.5.1/docs/Data-Containers-ListUtils.html#v:nubOrd just because `containers` comes with GHC (and we already depend on it). Also, we might as well fuse the `fmap`:

```suggestion
  inputDrvPaths <- uniqueSort . Prelude.lines <$> readProcess "nix-instantiate" [ jobsExpr ] ""
```

Reply:
No, I did not choose it specifically and felt bad for dragging in the dependency. I just wasn't aware of `nubOrd`, so your suggested change is an improvement all around IMO.

Reply:
Great! If you can make a change to `nubOrd`, I'll go ahead and merge this. Thanks!

Reply:
I'll try to get to this over the weekend and then rebase the other PRs I've made that are dependent on this one.

Reply:
Awesome, thank you! Sorry it's taking me a while to get to this, but I'm sure we can get these all in soon.

Reply:
No worries. I did not have time to do this over the weekend but it's still on my todo shortlist :)